blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/Serializers.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright 2013 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.io.serializers;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.netflix.astyanax.serializers.AbstractSerializer;
+import com.rackspacecloud.blueflood.exceptions.SerializationException;
+import com.rackspacecloud.blueflood.io.serializers.astyanax.HistogramSerializer;
+import com.rackspacecloud.blueflood.io.serializers.metrics.*;
+import com.rackspacecloud.blueflood.types.*;
+
+import java.nio.ByteBuffer;
+
+public class Serializers {
+    // NumericSerializer can be used with Rollup and full resolution metrics.
```

Comment:
if you want to get picky you can make a private constructor to make sure an instance is not created, but not required.